### ISSUE TITLE
How StaticLint.jl server is different than Lint.jl

### ISSUE THREAD

**Issue by aminya** [julia-vscode/StaticLint.jl#37]
I recently updated linter-julia for Atom: https://github.com/AtomLinter/linter-julia
It uses Lint.jl as the backend. However, we want to use StaticLint.jl instead as Lint.jl was made for older versions of the Julia. [issue](https://github.com/AtomLinter/linter-julia/issues/12)

inter-julia calls Lint.jl [API](https://github.com/tonyhffong/Lint.jl/blob/01fd4f11072d1940e6504b77117f49c72db70a27/src/server.jl#L105) through an API like [this](https://github.com/AtomLinter/linter-julia/blob/72453484949efe540fd24f68eb70da40d6e38d2a/lib/julia-server.jl#L30):
```julia
lintserver(named_pipe,"standard-linter-v2"); # named_pipe being the Julia exe
```

How is StaticLint.jl different than Linit.jl?

**Comment by davidanthoff** [comment]
StaticLint.jl is a linter based on CSTParser.jl, that powers the Julia VS Code extension. You can use it via LanguageServer.jl, but at least in theory you should also be able to use this package directly if you just want the linting story. In practice we don’t have a pretty API, I think. My best guess is that @ZacLN could create a simple API without too much work, but he’s been quite busy lately...

**Comment by aminya** [comment]
Is there any documentation or any sort of example for the API? 
We already have a server that can feed out and get an answer. It is implemented here: https://github.com/AtomLinter/linter-julia/blob/72453484949efe540fd24f68eb70da40d6e38d2a/lib/index.js#L75

Other than being in deps of LanguageServer, I couldn't find anywhere in the julia-vscode extension code, that StaticLint.jl is found (https://github.com/julia-vscode/julia-vscode/search?q=staticlint&unscoped_q=staticlint)

I hoped we had some documentation or examples somewhere, as a lot of effort has been made to make all of these different repositories, but they are only limitted to vs-code.

**Comment by davidanthoff** [comment]
No, unfortunately not... Yes, the reason for splitting these things into repos was that other could use them outside VS Code, and I'm more than aware that that is blocked right now by the lack of both an API and docs.  I wish I could give you some timeline, but I can't right now.

**Comment by schlichtanders** [comment]
any update on this? Please, this is really one of the most valuable additions to the julia ecosystem in total
